examples/frigate/frigate.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+from datetime import datetime
+
+try:
+    import json
+except ImportError:
+    import simplejson as json
+
+def frigate_events(topic, data, srv=None):
+    if srv is not None:
+        srv.logging.debug('events ******************************************')
+    a = json.loads(data['payload'])['after']
+    f = lambda x: (y.replace('_', ' ') for y in x)
+    r = {
+      'camera': a['camera'],
+      'label':  a['sub_label'] or a['label'],
+      'current_zones': ', '.join(f(a['current_zones'])),
+      'entered_zones': ', '.join(f(a['entered_zones'])),
+      'time':   datetime.fromtimestamp(a['frame_time'])
+    }
+    r.update({
+      'title':  f"{r['label']} entered {r['entered_zones']}",
+      'format': f"In zones {r['current_zones']} at {r['time']}",
+      'click':  f"https://frigate/events?camera={r['camera']}&label={r['label']}&zone={a['entered_zones'][0]}"
+    })
+    return r
+
+def frigate_events_filter(topic, message, section, srv=None):
+    if srv is not None:
+        srv.logging.debug('filter ******************************************')
```

Comment:
You can probably remove these debug messages.

Reply:
Addressed with c2623646b, thanks.